mixnet/node/src/lib.rs

```diff
@@ -42,6 +42,8 @@ impl MixnetNode {
     const CLIENT_NOTI_CHANNEL_SIZE: usize = 100;
 
     pub async fn run(self) -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+        tracing::info!("Public key: {:?}", self.public_key());
```

Comment:
nit: maybe debug?

Reply:
I wanted to print this as info, so that node operator can get its public key and share it with other operators.